hawkbit-ddi-api/src/main/java/org/eclipse/hawkbit/ddi/json/model/DdiDeployment.java

```diff
@@ -100,9 +117,40 @@ public String getName() {
         }
     }
 
+    /**
+     * Status of the maintenance window for action.
+     */
+    public enum MaintenanceWindowStatus {
```

Comment:
Is the maintenance status necessary in the deployment base result? 
```
"deployment": {
        "download": "forced",
        "update": "forced",
        "maintenanceWindow": "AVAILABLE"
.....
......
```
Wouldn't be sufficient the `"update" : "skip"` and `"update" : "forced"` indicate the update should be installed? is there an advanatge or use-case to cover the extra `maintenanceWindow` indicator?

Reply:
hmm... i guess this will depend on the client side agent... since we do not pass the information about the exact time of the window and duration, not sure what client can do with this extra piece of information.... as of now it is serving more of an indicator that this particular update was created with a maintenance schedule..